src/components/job-definition-row.tsx

```diff
@@ -92,7 +92,7 @@ export function buildJobDefinitionRow(
   forceReload: () => void,
   trans: TranslationBundle,
   ss: SchedulerService,
-  handleApiError: (error: string) => void
+  handleApiError: (error: string | undefined) => void
```

Comment:
`handleApiError` accepts `undefined` because it's one of the expected inputs / error object can be `undefined` right? 

(If undefined is used to indicate absence of value and function does not necessary expects to receive undefined otherwise it I think it is better to use `null` to indicate an absence of value s.t. there is a clear differentiation vs 
`undefined` and function would not accept just non-existing/unset field)